[2024-09-22T16:37:47.408+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T16:37:47.415+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [queued]>
[2024-09-22T16:37:47.421+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [queued]>
[2024-09-22T16:37:47.422+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-22T16:37:47.429+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): store_user> on 2024-09-22 16:28:40.273444+00:00
[2024-09-22T16:37:47.433+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8576) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T16:37:47.435+0000] {standard_task_runner.py:72} INFO - Started process 8578 to run task
[2024-09-22T16:37:47.435+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2024-09-22T16:28:40.273444+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpq6iyzfvr']
[2024-09-22T16:37:47.436+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask store_user
[2024-09-22T16:37:47.461+0000] {task_command.py:467} INFO - Running <TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [running]> on host 0bbd6d2129ab
[2024-09-22T16:37:47.497+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='store_user' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T16:28:40.273444+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T16:28:40.273444+00:00'
[2024-09-22T16:37:47.498+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-22T16:37:47.498+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-22T16:37:47.498+0000] {logging_mixin.py:190} INFO - Current task name:store_user state:running start_date:2024-09-22 16:37:47.415536+00:00
[2024-09-22T16:37:47.498+0000] {logging_mixin.py:190} INFO - Dag name:user_processing and current dag run status:running
[2024-09-22T16:37:47.498+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T16:37:47.499+0000] {postgres.py:191} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',' , filename: /tmp/processed_user.csv
[2024-09-22T16:37:47.503+0000] {base.py:84} INFO - Retrieving connection 'postgres'
[2024-09-22T16:37:47.505+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-22T16:37:47.510+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T16:37:47.510+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, run_id=manual__2024-09-22T16:28:40.273444+00:00, execution_date=20240922T162840, start_date=20240922T163747, end_date=20240922T163747
[2024-09-22T16:37:47.513+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-22T16:37:47.514+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-22T16:37:47.514+0000] {logging_mixin.py:190} INFO - Dag name:user_processing queued_at:2024-09-22 16:37:41.340603+00:00
[2024-09-22T16:37:47.514+0000] {logging_mixin.py:190} INFO - Task hostname:0bbd6d2129ab operator:PythonOperator
[2024-09-22T16:37:47.536+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T16:37:47.545+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T16:37:47.546+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
